### ISSUE TITLE
error when modifying gradient

### ISSUE THREAD

**Issue by mchikina** [mlverse/torch#830]
model$linear1$weight$grad=... returns an error "Error in f(value) : unused argument (value)" but the values *are* modified.

**Comment by dfalbel** [comment]
Thanks for reporting! This is a bug. 
You can use something like this as a workaround:

```
library(torch)
x <- torch_tensor(1)
x$set_grad_(torch_tensor(1))
x$grad
```

**Comment by dfalbel** [comment]
Will be fixed in #832 
Note that the bug was also reported in stack overflow: https://stackoverflow.com/questions/72032384/cant-modify-the-grad-of-a-tensor-in-r-torch
